Add support for Greek character set

blakestack opened the pull request:
<!-- Enter a brief description/summary of your PR here. What does it fix/what does it change/how was it tested (even manually, if necessary)? -->
## Summary of the Pull Request
This PR will add the Greek Charset to the Languages.cs file of PowerAccent. I tried to map each Greek letter to what I thought was the most appropriate English Character. I am not sure if this aligns with the current standard. The change has not been tested yet. I am seeking feedback on the current changes, and looking for help regarding the next steps and testing. 

<!-- Please review the items on the PR checklist before submitting-->
## PR Checklist

- [x] **Closes:** #24860
- [x] **Communication:** I've discussed this with core contributors already. If work hasn't been agreed, this work might be rejected
- [ ] **Tests:** Added/updated and all pass
- [ ] **Localization:** All end user facing strings can be localized
- [ ] **Dev docs:** Added/updated
- [ ] **New binaries:** Added on the required places
   - [ ] [JSON for signing](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ESRPSigning_core.json) for new binaries
   - [ ] [WXS for installer](https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/Product.wxs) for new binaries and localization folder
   - [ ] [YML for CI pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/ci/templates/build-powertoys-steps.yml) for new test projects
   - [ ] [YML for signed pipeline](https://github.com/microsoft/PowerToys/blob/main/.pipelines/release.yml)
- [x] **Documentation updated:** If checked, please file a pull request on [our docs repo](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/hub/powertoys) and link it here: https://github.com/MicrosoftDocs/windows-dev-docs/pull/4622

<!-- Provide a more detailed description of the PR, other things fixed or any additional comments/features here -->
## Detailed Description of the Pull Request / Additional comments

Seeking feedback for organizing the Greek characters to align with current PowerAccent standard.

<!-- Describe how you validated the behavior. Add automated tests wherever possible, but list manual validation steps taken as well -->
## Validation Steps Performed

Jay-o-Way commented:
@damienleroy this is one more case of the known discussion: Is Quick Accent designed to be about accents, or to "translate" between languages?

Aaron-Junker reviewed (changes requested):
In my opinion the lowercase letters should be displayed normally and the uppercase ones only when using shift.

They get automatically transformed in their uppercase variant anyway when Quick Accent is invoked with shift.

blakestack commented:
Thanks for the feedback so far. For the Greek charset would a better direction be to include only the lowercase accented Greek characters, as well as to remove the uppercase Greek characters from the current PR? I am not able to manually compile/test as yet, but I am unclear on the use of the shift key for PowerAccent.

Aaron-Junker commented:
https://github.com/microsoft/PowerToys/assets/58633848/7860dc1c-6c07-460c-850a-e14e7b6b972f

@blakestack When you hold shift while activating Quick Accent all characters will automatically be transformed into uppercase letters, so there is no need to include both.

> I am not able to manually compile/test as yet

What problems are you encountering?

htcfreek commented:
@Aaron-Junker 
Can you restart policy service pipeline?

Aaron-Junker commented:
> @Aaron-Junker 
> Can you restart policy service pipeline?

I can't or I don't know how

blakestack commented:
@Aaron-Junker Thanks for clarifying. I do not have my development environment set up correctly, but I committed some changes to the current PR. I removed the uppercase characters and added all the accented characters in the Greek alphabet.

blakestack commented:
@microsoft-github-policy-service agree

damienleroy commented:
> @damienleroy this is one more case of the known discussion: Is Quick Accent designed to be about accents, or to "translate" between languages?

Yes, that was the purpose (the name of the tool speak by himself). Going behind this, will made of this tool an another tool, with not the same kind of issue. Problem will be more complex to manage.

blakestack commented:
@Jay-o-Way @damienleroy With issue #24860, After reviewing the PR and comparing it to the current implementation, I think that PowerAccent/QuickAccent provides the accented or punctuated characters from each charset. I am not completely sure how that would look for the Greek alphabet as far as including all characters and accented characters, or only the accented characters. How do you recommend assigning each letter/symbol to a key or number as well?

Jay-o-Way commented:
Why don't you link `lambda` with <kbd>L</kbd> ?

blakestack commented:
@Jay-o-Way I started based on the shape of each character for my initial commit. Using the spelling of each character would be better, the only potential conflicts I can think of would be when assigning the letter O for `Omicron` or `Omega` and conflict for `Eta`/`Epsilon`.

blakestack commented:
I committed new changes with the Greek characters assigned to each corresponding letter key. I combined the Greek characters that were overlapping. 

Combined: 
- `Epsilon`/ `Eta`
- `Theta`/ `Tau`
- `Omicron`/ `Omega`
- `Pi`/ `Phi`/ `Psi`

blakestack commented:
@damienleroy @Aaron-Junker Can I get a review/feedback on the current changes?

blakestack commented:
Attempted to rename local and remote branches to a new name, and the pr closed

blakestack commented:
Note: `git branch -m "old-name" "new-name"` only renames the local branch.
